colorama/ansi.py

```diff
@@ -100,3 +101,33 @@ class AnsiStyle(AnsiCodes):
 Back   = AnsiBack()
 Style  = AnsiStyle()
 Cursor = AnsiCursor()
+
+fore_values = set(Fore.__dict__.values())
+back_values = set(Back.__dict__.values())
+style_values = set(AnsiStyle.__dict__.values())
+
+fore_stack = [Fore.RESET]
+back_stack = [Back.RESET]
+style_stack = [Style.NORMAL]
+
+@contextlib.contextmanager
+def colorama(*args):
```

Comment:
I think a better name would be `colorama_color` or something else. The name `colorama` is the name of the package and I don't think it's a good idea to override it. Also, we already have a `colorama_text` context manager which is something different, so it's better to avoid confusion.